src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -85,6 +101,21 @@ const std::map<std::string, std::string> GetModuleDataToCapabilitiesMapping() {
   mapping["signalChangeThreshold"] = "signalChangeThresholdAvailable";
   mapping["radioEnable"] = "radioEnableAvailable";
   mapping["state"] = "stateAvailable";
+  mapping["sisData"] = "sisDataAvailable";
+
+  // audio
+  mapping["source"] = "sourceAvailable";
+  mapping["keepContext"] = "sourceAvailable";
```

Comment:
I think adding a `keepContextAvailable` to the capabilities in mobile api and hmi api xml might be a good way to resolve this
mapping["keepContext"] = "keepContextAvailable";

+ <struct name="AudioControlCapabilities">
+     <param name="moduleName" type="String" maxlength="100" mandatory="true">
+         <description>
+         The short friendly name of the light control module.
+         It should not be used to identify a module by mobile application.
+         </description>
+     </param>
+     <param name="sourceAvailable" type="Boolean" mandatory="false">
+         <description>Availability of the control of audio source. </description>
+     </param>
+     <param name="keepContextAvailable" type="Boolean" mandatory="false">
+         <description>Availability of the keepContext paramter. </description>
+     </param>
+     <param name="volumeAvailable" type="Boolean" mandatory="false">
+         <description>Availability of the control of audio volume.</description>
+     </param>
+     <param name="equalizerAvailable" type="Boolean" mandatory="false">
+         <description>Availability of the control of Equalizer Settings.</description>
+     </param>
+     <param name="equalizerMaxChannelId" type="Integer" minvalue="1" maxvalue="100" mandatory="false">
+         <description>Must be included if equalizerAvailable=true, and assume all IDs starting from 1 to this value are valid</description>
+     </param>
+ </struct>